src/pages/dashboard/registration.tsx

```diff
@@ -578,18 +658,56 @@ function Registration() {
         </FormRow>
         <FormRow>
           <Dropdown
-            name='feedback'
+            name='willingToInterview'
             label='Select a response'
             options={[
               {label: 'Yes', value: true},
               {label: 'No', value: false},
             ]}
-            required={userForm.feedback.required}
-            value={userForm.feedback.value}
+            required={userForm.willingToInterview.required}
+            value={userForm.willingToInterview.value}
             onClick={handleDropdownChange}
             padded
           />
```

Comment:
Something I saw earlier was that this defaults to "No". Is this the desired behaviour?
Not sure if it changed with this PR either.

Reply:
I suppose we should default it to yes, doesn't make a difference

Reply:
Defaulting to yes isn't good UX. It shouldn't have a value at all by default.